### DISCUSSION TITLE
[blocks]: Sidebar 9 behaves quite strangely on mobile

### DISCUSSION THREAD

**Discussion by robsyc** [shadcn-ui/ui#5904] [Blocks Request]
### Description

Hey, I'm loving the sidebar blocks! For my project I'm really interested in [sidebar 9](https://next.shadcn-svelte.com/blocks#sidebar-09) in which there is a toolbar with icons, a collapsible section (who's content is adaptive) and the main content section.

However it behaves quite strangely on mobile. Namely, the collapsible section is completely hidden at all times. I think a design like this would work best on mobile if the toolbar with icons were placed on the bottom of the screen and the collapsible section were to pop-in from the bottom, along with a collapse button. I'll include a little sketch below.

![image](https://github.com/user-attachments/assets/c8a8164f-67b2-4718-986a-6721b4fa0489)

I'm not sure how difficult it would be to implement this 'vertical toolbar on desktop' but 'horizontal on mobile' behavior. I'll start playing around with it but I'd love to see what the community thinks.

**Comment by kabeerx9** [comment]
Did you implement this ?

**Comment by alzen8vtg** [comment]
shadcn team, any feedback for this sidebar-09's mobile view? as im facing same issue on this as well when mobile view, unable to access nested sidebar during Mobile View
